lib/constants/timezones.js

```diff
@@ -1,41 +1,94 @@
 import moment from 'moment-timezone';
+import { Map } from 'immutable';
 
 /**
  * Etc/GMT±HH type timezones
- * key is timezone name and value i
+ * key is timezone name and value is label
  *
  * @type {string[][]}
  */
-const UTC_BASED_ZONES = [
-  ['Etc/GMT+12', 'UTC -12:00'],
-  ['Etc/GMT+11', 'UTC -11:00'],
-  ['Etc/GMT+10', 'UTC -10:00'],
-  ['Etc/GMT+9', 'UTC -09:00'],
-  ['Etc/GMT+8', 'UTC -08:00'],
-  ['Etc/GMT+7', 'UTC -07:00'],
-  ['Etc/GMT+6', 'UTC -06:00'],
-  ['Etc/GMT+5', 'UTC -05:00'],
-  ['Etc/GMT+4', 'UTC -04:00'],
-  ['Etc/GMT+3', 'UTC -03:00'],
-  ['Etc/GMT+2', 'UTC -02:00'],
-  ['Etc/GMT+1', 'UTC -01:00'],
+const UTC_BASED_ZONE_TZ = [
+  ['-12:00', 'Etc/GMT+12'],
```

Comment:
Is this backwards, should it be `['-12:00', 'Etc/GMT-12']`? Notice the `+` (plus) that I've swapped for a `-` (minus).

Reply:
It's confusing point, but Etc/GMT+12 is correct.

- https://en.wikipedia.org/wiki/List_of_tz_database_time_zones
- https://askubuntu.com/questions/519550/why-is-the-8-timezone-called-gmt-8-in-the-filesystem

I'll add comment.

Reply:
```
> var m = moment();
undefined
> m
moment("2019-05-24T12:03:11.027")
> m.tz('Etc/GMT+12')
moment.parseZone("2019-05-23T23:03:11.027-12:00")
> m.tz('Etc/GMT-12')
moment.parseZone("2019-05-24T23:03:11.027+12:00")
```

Reply:
Oh wow nice, that's very odd, thanks for the code example 👍